src/components/AddTransaction/index.js

```diff
@@ -38,12 +40,16 @@ export default function AddTransaction() {
 					transaction.change = parseFloat(transaction.change);
 					if (transactionID) {
 						editTransaction(transactionID, transaction);
-						alert('edited');
+						notifyEdit();
 					} else {
 						addTransaction(accountID, transaction);
+						notifyAdd();
+						console.log('added');
```

Comment:
Please remove `console.log`s before merging into main :)
```suggestion
```

Reply:
Thanks, I wasn't aware this was still in the code! 😂